Services/UI/OJS.Services.Ui.Data/IParticipantScoresDataService.cs

```diff
@@ -11,6 +11,8 @@ public interface IParticipantScoresDataService : IDataService<ParticipantScore>
     {
         Task<ParticipantScore?> GetByParticipantIdAndProblemId(int participantId, int problemId);
 
+        IQueryable<ParticipantScore> GetByProblemIdAndParticipants(IEnumerable<int> participantIds, int problemId);
```

Comment:
Returning `IQueryable` from services in not great. `IQueryable` should be internal  only